lms/envs/common.py

```diff
@@ -4145,10 +4102,8 @@
 CUSTOM_RESOURCE_TEMPLATES_DIRECTORY = None
 
 # API access management
-API_ACCESS_MANAGER_EMAIL = 'api-access@example.com'
-API_ACCESS_FROM_EMAIL = 'api-requests@example.com'
-API_DOCUMENTATION_URL = 'https://course-catalog-api-guide.readthedocs.io/en/latest/'
-AUTH_DOCUMENTATION_URL = 'https://course-catalog-api-guide.readthedocs.io/en/latest/authentication/index.html'
+API_ACCESS_FROM_EMAIL = None
+API_ACCESS_MANAGER_EMAIL = None
```

Comment:
It seems that LMS will not start if these are set to None, so this change breaks tutor dev. Could you restore these to  their old value? 
```suggestion
API_ACCESS_MANAGER_EMAIL = 'api-access@example.com'
API_ACCESS_FROM_EMAIL = 'api-requests@example.com'
```

(They're not very good default values, but this will un-stick this PR, and maybe in the future we can come back later and improve them to something better like `Derived(lambda settings: settings.DEFAULT_FROM_EMAIL)`.)

Reply:
I've made this change, which changes the diff settings output in this way:
```diff
diff main_settings_dump/lms.envs.production__lms_envs_minimal.json feature_settings_dump/lms.envs.production__lms_envs_minimal.json
852c852
<     "API_ACCESS_FROM_EMAIL": null,
---
>     "API_ACCESS_FROM_EMAIL": "api-requests@example.com",
```

Reply:
Looks good.